docs/api/geo_datasets.csv

```diff
@@ -1,17 +1,18 @@
 Dataset,Type,Source,Size (px),Resolution (m)
-`Aboveground Woody Biomass`_,Mask,"Landsat, LiDAR","~40,000x40,000",~30
+`Aboveground Woody Biomass`_,Mask,"Landsat, LiDAR","40,000x40,000",30
 `Aster Global DEM`_,Mask,Aster,"3,601x3,601",30
-`Canadian Building Footprints`_,Labels,Generated,,
-`Chesapeake Land Cover`_,"Imagery, Labels",,,1
-`Global Mangrove Distribution`_,Mask,Generated,,3
-`Cropland Data Layer`_,Labels,Aerial,,
-`EDDMapS`_,Labels,Vector,,
-`EnviroAtlas`_,"Imagery, Labels",Aerial,,1
-`Esri2020`_,Labels,Sentinel-2,,10
-`EU-DEM`_,Labels,"Aster, SRTM, Russian Topomaps",,25
-`GBIF`_,Labels,Vector,,
-`GlobBiomass`_,Labels,Landsat,"45,000x45,000",~100
-`Landsat`_,Imagery,Landsat,,30
-`NAIP`_,Imagery,Aerial,,1
-`Open Buildings`_,Labels,Generated,,
-`Sentinel`_,Imagery,Sentinel,,10
+`Canadian Building Footprints`_,Geometries,Bing Imagery,-,-
+`Chesapeake Land Cover`_,"Imagery, Mask",NAIP,-,1
+`Global Mangrove Distribution`_,Mask,"Remote Sensing, In Situ Measurements",-,3
+`Cropland Data Layer`_,Mask,Aerial,-,30
+`EDDMapS`_,Points,Citizen Scientists,-,-
+`EnviroAtlas`_,"Imagery, Mask","NAIP, NLCD, OpenStreetMap",-,1
+`Esri2020`_,Mask,Sentinel-2,-,10
+`EU-DEM`_,Mask,"Aster, SRTM, Russian Topomaps",-,25
+`GBIF`_,Points,Citizen Scientists,-,-
+`GlobBiomass`_,Mask,Landsat,"45,000x45,000",100
+`iNaturalist`_,Points,Citizen Scientists,-,-
```

Comment:
iNaturalist was missing from the table.

For GBIF/EDDMapS/iNaturalist I used "points" for the dataset type and "citizen scientists" for the label sources